docs/api.md

```diff
@@ -2586,6 +2593,14 @@ console.log(`Bytes used: ${usedBytes / totalBytes * 100}%`);
 > **NOTE** JavaScript Coverage doesn't include anonymous scripts. However, scripts with sourceURLs are
 reported.
 
+### class: Worker
+
+#### worker.executionContext()
```

Comment:
One sentence description would be nice. I assume this covers service workers too? 

"Worker provides an API for interacting with the web and service workers".

Reply:
I’m holding off on writing the docs until everything has been sufficiently bikeshedded. I’m not sure how to handle service workers yet.